server/testing/src/main/java/org/apache/james/utils/UpdatableTickingClock.java

```diff
@@ -38,7 +38,7 @@ public void setInstant(Instant instant) {
 
     @Override
     public ZoneId getZone() {
-        throw new NotImplementedException("No timezone attached to this clock");
+        return ZoneId.systemDefault();
```

Comment:
you should never rely on the system defaults, being for ZoneId, Charset or anything else. It's the best way to have various failures at deploy time.

Reply:
please use systemUTC instead